03_classification.ipynb

```diff
@@ -2427,35 +2436,36 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "By simply augmenting the data, we got a 0.5% accuracy boost. :)"
+    "Chỉ với việc tăng cường dữ liệu, độ chính xác đã tăng 0.5%. :)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## 3. Tackle the Titanic dataset"
+    "## 3. Làm việc với tập dữ liệu Titanic"
```

Comment:
```suggestion
    "## 3. Giải quyết tập dữ liệu Titanic"
```

Reply:
Em thấy `giải quyết` nghe hơi mạnh, nếu đi với `vấn đề` hay `bài toán` em mới dùng từ này.